src/Veldrid.SDL2/Sdl2.SysWMInfo.cs

```diff
@@ -43,7 +43,7 @@ public struct Win32WindowInfo
     public struct X11WindowInfo
     {
         public IntPtr display;
-        public IntPtr Sdl2Window;
+        public IntPtr window;
```

Comment:
For better or worse, these typos are part of the public API, and can't be changed at this point.

Reply:
Okay, would you like me to revert the variable names?